### ISSUE TITLE
RESUB 1.0.4 sim error (reoccuring sims)

### ISSUE THREAD

**Issue by Gareld-Michael** [xXXIMMATTHEWXXx/Sim-u-kraft-Restored#1]
Hey there, I got your email, I will recreate the issue. (I am glad your going to try to redo S-U-K)
The issue appeared to be that when the new world was made, the old deleted world that I had just been playing on (so no Minecraft restart) crossed with the new one and old people from the older deleted world appeared in THE EXACT SAME ORDER (I think I remember) in the new world except the beginning person who I believe also mysteriously disappeared after older characters began to appear with in-fact no houses yet built. The comments that appear here but not in the original comment are things that I believe I remember, but being a year or two ago I am not so sure exactly. Use those as clues (potentially)

THE ORIGINAL TEXT BELOW - Only respelled the name _Minecraft_
When I was playing 1.0.4 in a new world a Sim spawned in, like normal. About 20 minutes of play later (I was trying to find the sim-u-constructor recipe an did other normal stuff) I entered creative got some sim-u-stuff (not literally an object) (sim-u : constructor, miner, farmer + 4 markers) and tried to get my sim to build a house after I reentered survival. When I tried to hire her, other names from a another save that had been removed, because of a problem, appeared with hers and when I tried to hire her and another one of them later, neither one would come. Even after I chose a building they did not come there to build it. I'm thinking as I'm typing this that it may be a problem from another save in a different version that I didn't expect to carry over. I'll then reload _Minecraft_ and try again. However if there is a way to prevent any form of carry over I would ask you to try your best to remove it. Thanks for listening.

**Comment by xXXIMMATTHEWXXx** [comment]
Yes, ashjacks code states that he could not find a way around that. I am
currently looking into it

**Comment by Gareld-Michael** [comment]
I hope you find a way, that was one of the most annoying errors of the mod. Also, your right ,ashjack made plenty of statements of just that. Like I suggested in the next resubmission it could be a cache error or a forgotten code segment on unloading and reloading worlds. Basically he forgot to utilize an unload function that Minecraft already has.

**Comment by xXXIMMATTHEWXXx** [comment]
Yeah. I am working to contact him to get his source. His github was not up
to date
